app/(service)/my-store/my-recruits/edit/_components/registRecruitForm.tsx

```diff
@@ -0,0 +1,97 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+
+import Button from '@/components/button/button';
+import Input from '@/components/input/input';
+
+interface FieldValues {
+  hourlyPay: string;
+  registTime: string;
+  workTime: string;
+  description: string;
+}
+
+function formatHourlyPay(input: string) {
+  const onlyNums = input.replace(/[^0-9]/g, '');
+  return onlyNums.length < 4 ? onlyNums : onlyNums.replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,');
+}
+
+export default function RegistRecruitForm() {
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<FieldValues>({ mode: 'all' });
+
+  const hourlyPay = watch('hourlyPay', '');
+
+  const [displayHourlyPay, setDisplayHourlyPay] = useState('');
+
+  useEffect(() => {
+    setDisplayHourlyPay(formatHourlyPay(hourlyPay));
+  }, [hourlyPay]);
+
+  /**
+   * @todo 공고등록 로직 추가
+   */
+  const onSubmit = (data: FieldValues) => {
+    console.log(data);
+  };
+
+  return (
+    <form onSubmit={handleSubmit((data) => onSubmit(data))}>
```

Comment:
P3: 이렇게만 작성해주셔도 될 것 같아요!

```suggestion
    <form onSubmit={handleSubmit((onSubmit)}>
```